daemon/daemon.go

```diff
@@ -176,7 +176,7 @@ func operationalServerUp() {
 
 // operationalServerdown stops the operational server if it is running
 func operationalServerDown() {
-	if server.Running() == server.OPERATIONAL {
+	if server.Current == server.Operational && (server.State == server.Running || server.State == server.Starting) {
```

Comment:
why is a similar set of conditions not also needed inside managementServerDown()?